os/arch/arm/src/amebad/amebad_serial.c

```diff
@@ -417,7 +418,6 @@ static int rtl8721d_up_setup(struct uart_dev_s *dev)
 	serial_baud(sdrv[uart_index_get(priv->tx)], priv->baud);
 	serial_format(sdrv[uart_index_get(priv->tx)], priv->bits, priv->parity, priv->stopbit);
 	serial_set_flow_control(sdrv[uart_index_get(priv->tx)], priv->FlowControl, priv->rts, priv->cts);
-	serial_enable(sdrv[uart_index_get(priv->tx)]);
```

Comment:
The only difference in changes is removing this line. Other changes are redundant code or tidy up the code. 
Is it okay we don't have this line?

Reply:
Yes, the corresponding peripheral clock functions are enabled in the bootloader, therefore it is redundant to enable again at here. Thanks!

Reply:
Ok. Thanks for checking. I will approve it.